test/api-endpoints/sequelize/balance.routes.sequelize.spec.ts

```diff
@@ -0,0 +1,226 @@
+        .set("content-type", "application/json")
+        .send(balanceWithInvalidName)
+        .end((err, res) => {
+          expect(res.status).to.equal(500);
+          done(err);
+        });
+    });
+
+    it("should not create a balance with invalid cryptocurrency", (done) => {
+      chai
+        .request(app)
+        .post(`/sequelize/balance/create/${balanceTesting.client}`)
+        .set("content-type", "application/json")
+        .send(balanceWithInvalidCryptocurrency)
+        .end((err, res) => {
+          expect(res.status).to.equal(500);
+          done(err);
+        });
+    });
+  });
+
+  describe(`/sequelize/balance/get/${balanceTesting.client}/${balanceTesting.cryptocurrency}`, () => {
+    it("should show a balance if exists", (done) => {
+      chai
+        .request(app)
+        .get(
+          `/sequelize/balance/get/${balanceTesting.client}/${balanceTesting.cryptocurrency}`
+        )
+        .set("content-type", "application/json")
+        .send(balanceTesting)
+        .end((err, res) => {
+          const { client, amount, cryptocurrency } = res.body;
+          const requestedBalance = {
+            client: client,
+            amount: amount,
+            cryptocurrency: cryptocurrency,
+          };
+          expect(requestedBalance).to.be.deep.equal(balanceTesting);
+          done(err);
+        });
+    });
+
+    it("shouldn't return a balance if the client doesn't exist", (done) => {
+      chai
+        .request(app)
+        .get(
+          `/sequelize/balance/get/${randomString(39)}/${
+            balanceTesting.cryptocurrency
+          }`
+        )
+        .set("content-type", "application/json")
+        .send(balanceTesting)
+        .end((err, res) => {
+          expect(res.status).to.equal(404);
+          done(err);
+        });
+    });
+  });
+
+  describe(`/sequelize/balance/update/${balanceTesting.client}/${balanceTesting.cryptocurrency}`, () => {
+    it("should update a balance if exists", (done) => {
+      chai
+        .request(app)
+        .patch(
+          `/sequelize/balance/update/${balanceTesting.client}/${balanceTesting.cryptocurrency}`
+        )
+        .set("content-type", "application/json")
+        .send(balanceTesting2)
+        .end((err, res) => {
+          expect(res.status).to.equal(204);
+          done(err);
+        });
+    });
+  });
+
+  describe(`/sequelize/balance/get/${balanceTesting.client}/${balanceTesting.cryptocurrency}`, () => {
+    it("should show the updated balance", (done) => {
+      chai
+        .request(app)
+        .get(
+          `/sequelize/balance/get/${balanceTesting.client}/${balanceTesting.cryptocurrency}`
+        )
+        .set("content-type", "application/json")
+        .send(balanceTesting)
+        .end((err, res) => {
+          const { client, amount, cryptocurrency } = res.body;
+          const requestedBalance = {
+            client: client,
```

Comment:
You can avoid putting the same name twice:
cliente: client, ---> client,
Same with the other attributtes